src/components/proposalList.jsx

```diff
@@ -87,11 +87,14 @@ const ProposalsList = ({ proposals, customTerms }) => {
       );
       return;
     }
+    const isActiveFilter = option?.value === "Action Needed";
     searchMode.current = false;
     setFilter(option);
+    setSort({ name: isActiveFilter ? "Oldest" : "Newest", value: `submissionDate${isActiveFilter ? "Asc" : "Desc"}` })
   };
 
   const handleSort = (option) => {
+    console.log(option)
```

Comment:
drop this :-)

Reply:
Lol, sorry .. But this has been removed in the upcoming PR